### DISCUSSION TITLE
Can Someone tell me what im doing wrong?

### DISCUSSION THREAD

**Discussion by BlakVixen02** [Sehelitar/Kick.bot#62] [General]
Hi! Im trying to use this for the first time. Im on Streamer.bot (0.2.3) and have been using it without too many issues for a while. I got the newest version of the Kick.bot. Imported. Moved files to dll. On the subaction i added the references kick.dll and kick.bot.dll. i compile and it says it was successful. But when i hit OK, no kick login pops up and Kick does not show up in my trigger options. ive tried moving the files out of the dll folder but i still have the same issue. 
[log_20240715.log](https://github.com/user-attachments/files/16241738/log_20240715.log)

**Comment by Sehelitar** [comment]
Hey! 👋 
This log seems to indicate that the 2 DLLs are missing:
```
[2024-07-15 19:24:17.055 ERR] Unable to locate reference: D:\Stream Stuff\Streamer Bot\dlls\Kick.Bot\Kick.dll
[2024-07-15 19:24:17.055 ERR] Unable to locate reference: D:\Stream Stuff\Streamer Bot\dlls\Kick.Bot\Kick.bot.dll
```
Have all DLLs from the archive been copied in that folder?
